docs/resources/apig_throttling_policy.md

```diff
@@ -0,0 +1,137 @@
+  This parameter applies with each of the API call limits: `max_api_requests`, `max_app_requests`, `max_ip_requests`
+  and `max_user_requests`.
+
+* `max_api_requests` - (Required, Int) Specifies the maximum number of times an API can be accessed within a specified
+  period.
+  The value of this parameter cannot exceed the default limit 200 TPS.
+
+* `type` - (Optional, String) Specifies the type of the request throttling policy.
+  The valid values are as following:
+  - API-based: limiting the maximum number of times a single API bound to the policy can be called within the specified
+  period.
+  - API-shared: limiting the maximum number of times all APIs bound to the policy can be called within the specified
+  period.
+
+* `description` - (Optional, String) Specifies the description about the API throttling policy.
+  The description contain a maximum of 255 characters and the angle brackets (< and >) are not allowed.
+  Chinese characters must be in UTF-8 or Unicode format.
+
+* `period_unit` - (Optional, String) Specifies the time unit for limiting the number of API calls.
+  The valid values are *SECOND*, *MINUTE*, *HOUR* and *DAY*, default to *MINUTE*.
+
+* `max_app_requests` - (Optional, Int) Specifies the maximum number of times the API can be accessed by an app within
+  the same period.
+  The value of this parameter must be less than or equal to the value of `max_user_requests`.
+
+* `max_ip_requests` - (Optional, Int) Specifies the maximum number of times the API can be accessed by an IP address
+  within the same period.
+  The value of this parameter must be less than or equal to the value of `max_api_requests`.
+
+* `max_user_requests` - (Optional, Int) Specifies the maximum number of times the API can be accessed by a user within
+  the same period.
+  The value of this parameter must be less than or equal to the value of `max_api_requests`.
+
+* `user_throttles` - (Optional, List) Specifies an array of one or more special throttling policies for IAM user limit.
+  The `throttle` object of the `user_throttles` structure is documented below.
+
+* `app_throttles` - (Optional, List) Specifies an array of one or more special throttling policies for APP limit.
+  The `throttle` object of the `user_throttles` structure is documented below.
+
+The `throttle` block supports:
+
+* `max_api_requests` - (Required, Int) Specifies the maximum number of times an API can be accessed within a specified
+  period.
+
+* `throttling_object_id` - (Required, String) Specifies the object ID which the special throttling policy belongs.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - ID of the API throttling policy.
+* `user_throttles/throttling_object_name` - The object name which the special user throttling policy belongs.
+* `user_throttles/id` - ID of the special user throttling policy.
```

Comment:
Suggest to write in the following format. `app_throttles` is the same.

* `user_throttles` - Specifies an array of one or more special throttling policies for APP limit.
    Each item of data is as follows:
    -  `throttling_object_name` - The object name which the special user throttling policy belongs.
    -  `id` - ID of the special user throttling policy.

Reply:
These texts are more readable.